<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MouseColorGui</class>
 <widget class="QDialog" name="MouseColorGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxGetSkin">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>98</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Skin</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MouseColorGui</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
